docs/cluster.md

```diff
@@ -0,0 +1,254 @@
+# Running kubeaudit in a Cluster
+
+Kubeaudit can be run in a Kubernetes cluster by using the [official Docker image](https://hub.docker.com/r/shopify/kubeaudit): `shopify/kubeaudit`.
+
+## Without RBAC
+
+Example Job configuration:
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kubeaudit
+  namespace: default
+spec:
+  template:
+    metadata:
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/kubeaudit: runtime/default
+        seccomp.security.alpha.kubernetes.io/pod: runtime/default
+    spec:
+      automountServiceAccount: false
```

Comment:
should we also specifiy `RestartPolicy` ? I ran into some issues because I didn't specify it...